portal/src/RoleAndGroupsLayout.tsx

```diff
@@ -9,8 +10,13 @@ import {
   ErrorMessageBarContextProvider,
 } from "./ErrorMessageBar";
 
-interface RoleAndGroupsLayoutProps {
+export interface RoleAndGroupsHeaderProps {
   breadcrumbs: BreadcrumbItem[];
+  subitem?: JSX.Element | null;
```

Comment:
Use `React.ReactNode` instead of JSX.Element. Remove `/* global JSX */` at the top of this file as well.